locations/spiders/dacia_it.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+import scrapy
+
+from locations.dict_parser import DictParser
+
+
+class DaciaItSpider(scrapy.Spider):
+    name = "dacia_it"
+    item_attributes = {
+        "brand": "Dacia",
+        "brand_wikidata": "Q27460",
+    }
+    allowed_domains = ["dacia.it"]
+    start_urls = ["https://www.dacia.it/concessionario/lista-concessionari.data?page=0"]
+
+    def parse(self, response):
+        data = (
+            response.json()
+            .get("content", {})
+            .get("editorialZone", {})
+            .get("slice54vb", {})
+            .get("dealers", {})
+            .get("data")
+        )
+        for row in data:
+            item = DictParser.parse(row)
+            item["ref"] = row.get("birId")
+            item["street_address"] = row.get("address", {}).get("extendedAddress")
+            item["lat"] = row.get("geolocalization", {}).get("lat")
+            item["lon"] = row.get("geolocalization", {}).get("lon")
+            item["phone"] = row.get("telephone")
```

Comment:
The telephone looks like this:

```
"phone": {
    "type": "office",
    "value": "+390184298106"
  },
```

Can you pull just the phone number out of it?